packages/jest-config/src/normalize.ts

```diff
@@ -745,6 +745,52 @@ export default function normalize(
         }
         break;
       }
+      case 'displayName': {
+        const displayName = oldOptions[key] as Config.DisplayName;
+        const isDisplayNameAnObject =
```

Comment:
Using a modified version of this from stack overflow. 

https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript

Reply:
I think `jest-get-type` would cover this use case

Reply:
Oh didn't know about that small package. Thanks! Will amend :)